### PR TITLE
🐛 Fix cache paths

### PR DESCRIPTION
**Pull request by Koncopd**
Fix https://github.com/laminlabs/lamindb-setup/issues/868

### CODE REVIEW

File: docs/hub-prod/test-cloud-sync.ipynb
Comment (ap--) [inline docs/hub-prod/test-cloud-sync.ipynb:1]: shouldn't this just be?

```python
cloud_file.touch(exist_ok=True)  
```

i.e. do you really want the file to be potentially truncated to 0 bytes?

see: fsspec/universal_pathlib#262

Comment (Koncopd) [reply docs/hub-prod/test-cloud-sync.ipynb:1]: This one yes.
Without truncate is not implemented error.

Comment (Koncopd) [reply docs/hub-prod/test-cloud-sync.ipynb:1]: I can also do just `unlink` and `touch` ofc.

Comment (ap--) [reply docs/hub-prod/test-cloud-sync.ipynb:1]: This is caught in universal-pathlib==0.2.5 though. Am I missing an edge case with the touch behaviour here?

https://github.com/fsspec/universal_pathlib/blob/5486fa3c30cbb3f79637252d980994a1d738b0f5/upath/core.py#L970-L980

Comment (Koncopd) [reply docs/hub-prod/test-cloud-sync.ipynb:1]: Yes, yes, i mean in s3fs. But if i do via upath here it won't update the timestamp. This is the whole point of calling it here.

**Pull request merged**